### PR TITLE
Improve Air Flow Afterburn using 2x5015 fans

### PR DESCRIPTION
**Pull request by Simone67**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
yes
#### How was it tested? 
did it
#### Any background context you want to provide?
no
#### Screenshots (if appropriate)
Yes
#### Further notes
no

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ like so:
 
 | Creator | Mod title | Description | V0 | V1.8 | V2.4 | VSW | Trident |
 | --- | --- | --- | --- | --- | --- | --- | --- |
+|Simone67|[Tornado Air Flow](./Simone67/TornadoAirFlow)| Use two 5015 fan to get a super airflow|:x:|:x:|:heavy_check_mark:|:x:|:x:|
```
Comment (eduncan911) [inline printer_mods/README.md:28]: This should be moved down to it's position in Alphabetical ordering, not at the top.

Comment (eduncan911) [reply printer_mods/README.md:28]: You indicated this is only for V2.4, and not compatible with any other printer (with X).

Why wouldn't this work on any other printer that has an Afterburner? Like the V1.8, Trident, and Switchwire?

Comment (Simone67) [reply printer_mods/README.md:28]: Thanks for your comments, I will fix anyone..

Comment (Simone67) [reply printer_mods/README.md]: I did it..

File: printer_mods/Simone67/TornadoAirFlow/README.MD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Tested with recommended settings for Voron parts:Layer 0.2 - ABS, 4 perimeters, 5 tops/bottoms, 40% infill.
+All STL files are correctly oriented and do not need support.
+STL were tested by printing and assembling !!!
+  
+## Printing Orientation
+  ![one Side View](./IMAGE/orientation.jpg) 
+
+  ## Hardware
+  
+Qty | Item
```
Comment (bdbell) [inline printer_mods/Simone67/TornadoAirFlow/README.MD:29]: While Github appears to render this, please confirm to proper Markdown syntax and put the vertical lines at the start and end of each table row.

File: printer_mods/Simone67/TornadoAirFlow/README.MD [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<H1>TORNADO AIR FLOW for AfterBurner</H1><BR>
+  
+  ![one Side View](./IMAGE/sez_B.png)
+
+  Using two 5015 fans to get very powerful workpiece cooling air flow and necessary to get quality prints when using PLA material which needs strong air flow to cool immediately.
```
Comment (bdbell) [inline printer_mods/Simone67/TornadoAirFlow/README.MD:5]: "and" should be "are"

**Comment by bdbell** [comment]
How are the files STL/Main_Body_VORON_Clockwork_X Carriage.stl and STL/Toolhead_Cartridge_E3D_V6.stl any different than stock?

**Comment by cbc02009** [comment]
Closing due to inactivity.

**Pull request closed without merging**